crates/db/Cargo.toml

```diff
@@ -18,3 +18,5 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 tokio = { workspace = true }
 thiserror = { workspace = true }
+log = { version = "0.4", features = ["max_level_debug", "release_max_level_warn"] }
```

Comment:
hmm, why? 🤔 

I have the `tracing` crate on `main` btw, which is a replacement for this

Reply:
committed by mistake